src/components/About.jsx

```diff
@@ -21,7 +21,8 @@ function About() {
           <div className="">
             <span className="team p-1 fs-2 fw-bold ">API Used</span>
             <ul className="custom-list fs-6 fw-bold p-3">
-              <li>NewYork API</li>
+              <li>NewYorkTimes API</li>
```

Comment:
This is cool, do we have any references we should also add here? Not sure if we used any additional resources enough to reasonably document them.

Reply:
This should be Google Books API and New York Times Bestsellers API, but I can change this later.